nil_test.go

```diff
@@ -0,0 +1 @@
+package pack
```

Comment:
```suggestion
package pack_test
```
Some basic tests should be added.

Reply:
This was just a draft PR :)